Symfony/CS/Tests/Tokenizer/Transformer/BraceClassInstantiationTest.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+/*
+ * This file is part of the PHP CS utility.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Symfony\CS\Tests\Tokenizer\Transformer;
+
+use Symfony\CS\Tests\Tokenizer\AbstractTransformerTestBase;
+use Symfony\CS\Tokenizer\Tokens;
+
+/**
+ * @author Sebastiaans Stok <s.stok@rollerscapes.net>
+ */
+class BraceClassInstantiationTest extends AbstractTransformerTestBase
+{
+    /**
+     * @dataProvider provideProcessCases
+     */
+    public function testProcess($source, array $expectedTokens)
+    {
+        $tokens = Tokens::fromCode($source);
```

Comment:
copy-paste is wrong, isn't it?
You take provided data style from one test but test method from other.
please, use makeTest